bokehjs/src/coffee/models/scales/log_scale.coffee

```diff
@@ -2,24 +2,10 @@ import {Scale} from "./scale"
 import * as p from "core/properties"
 
 export class LogScale extends Scale
-
-  connect_signals: () ->
-    super()
-    @connect(@properties.source_range.change,       () -> @_state = null)
-    @connect(@properties.target_range.change,       () -> @_state = null)
-    @connect(@source_range.properties.start.change, () -> @_state = null)
-    @connect(@source_range.properties.end.change,   () -> @_state = null)
-    @connect(@target_range.properties.start.change, () -> @_state = null)
-    @connect(@target_range.properties.end.change,   () -> @_state = null)
-
-  @getters {
-    state: () ->
-      if not @_state? then @_state = @_compute_state()
-      return @_state
-  }
+  type: "LogScale"
 
   compute: (x) ->
-    [factor, offset, inter_factor, inter_offset] = @state
+    [factor, offset, inter_factor, inter_offset] = @_compute_state()
```

Comment:
I'm willing to believe the overhead is negligible when using `v_compute`. It's also worth noting that the cache is *currently* invalidated very often, every time there is any pan or zoom at all, e.g., and performance is fine. The code around this has change considerably so I'm also fine taking a step bak and doing the simple thing, and re-introducing caching with a fresh start later if it is determined to be necessary.